boards/arm/particle_argon_nrf52/board.cmake

```diff
@@ -0,0 +1,4 @@
+board_runner_args(nrfjprog "--nrf-family=NRF52")
```

Comment:
Is it weird that argon is setup for nrfjprog, but xenon is setup for pyocd?
https://github.com/zephyrproject-rtos/zephyr/pull/11587/commits/ac6d37d1eb885b3d82e67df1691d1c613bf46d2d#diff-80142bcc0090266663d3a1d6dfec6b50R1

Should we standardize on this?

Reply:
interesting.  @carlescufi did you have any preference on what runner is used for the particle.io boards?

Reply:
I'd like to see nrfjprog supported on xenon; I patch it that way in my dev branch.  Particle does sell a CMSIS-DAP board for programming these things, though, so pyocd is likely to be the primary interface.

Reply:
It really depends on what works out-of-the-box without having to reprogram the IC. Can someone let me know what is the default firmware in the debug chip on these boards? Is it Segger J-Link (used by `nrfjprog`) or CMSIS-DAP?

Reply:
@carlescufi nrfprog was working out of the box for me, pyocd didn't

Reply:
@mike-scott I think we should settle on J-Link because that is the "official" Nordic debugging mechanism.
@pabigot could you send a patch for the Xenon to support nrfjprog?

Reply:
@carlescufi well, no, because it hasn't been merged yet and needs rework.  I've pinged @ekarlso.

Reply:
Ok, nrfjprog it is.  I'll drop my -1 here.  Thanks for discussing. :+1: